tool/src/test/java/org/wildfly/security/tool/VaultCommandTest.java

```diff
@@ -229,16 +481,7 @@ public void bulkConversionMissingLocationOption() {
     @Test
     public void bulkConversionMissingEncryptionDirOption() {
         String[] args = {"--bulk-convert", "target/test-classes/bulk-vault-conversion-no-enc-dir"};
-        boolean testFailed = true;
-        try {
-            executeCommandAndCheckStatus(args);
-        } catch (Exception e) {
-            // Exception is wrapped inside ELYTOOL00012
-            Assert.assertTrue(e.getCause().getCause().getMessage().contains("ELYTOOL00022"));
-            testFailed = false;
-        }
-
-        Assert.assertFalse("Test was supposed to throw exception!", testFailed);
+        executeVaultCommandWithParams(args, false, "ELYTOOL00012");
```

Comment:
@nikitagibor Shouldn't the expectedOutput be "ELYTOOL00022" instead?

Reply:
I fixed it